Issue 8 FAQ page

tylerz1443 opened the pull request.

sanehirakenji reviewed (changes requested):
Hey Tyler, just checking. Is this the format you wanted this page to look like? I ran in my ide and this is how the page looks in my browser: 
<img width="1277" alt="FAQ" src="https://github.com/volunthree/volunthree/assets/97561440/12166a9b-0f53-4a6c-bbed-4a86d45a432c">

Could you also change the comment to fit the page as well as add routing to allow the user to access the page?

Thanks!

tylerz1443 commented:
ok think I fixed everything and I have updated the issue to the current version of main.  I have not added the page to the app.jsx page yet, but if both of these pages are fine I will added it to one of these pages for both of them and the push it as an update.
